streams/src/main/java/org/apache/kafka/streams/kstream/HoppingWindows.java

```diff
@@ -23,14 +23,16 @@
 import java.util.Map;
 
 /**
- * The hopping window specifications used for aggregations.
+ * The hopping window specifications used for aggregations. TODO extend
```

Comment:
The TODO was not meant to get merged. I was just a tag for this PR to get the reviewers attention, that I don't know how to improve this. I will use your suggestion :)